src/NHSD.BuyingCatalogue.Ordering.Api/Controllers/OrderItemsController.cs

```diff
@@ -63,7 +63,7 @@ public sealed class OrderItemsController : ControllerBase
             if (result.IsSuccess)
             {
                 createOrderItemResponse.OrderItemId = result.Value;
-                return CreatedAtAction(nameof(CatalogueSolutionsController.GetOrderItemAsync).TrimAsync(), "CatalogueSolutions", new { orderId, orderItemId = createOrderItemResponse.OrderItemId }, createOrderItemResponse);
+                return CreatedAtAction(nameof(GetAsync).TrimAsync(), null, new { orderId, orderItemId = createOrderItemResponse.OrderItemId }, createOrderItemResponse);
```

Comment:
Should the controller name be set here to the order items controller?

Reply:
Could be, if you provide null it assumes it's the same controller but I can explicitly provide it